documentation/decisions/0005-rdf-vocabulary.md

```diff
@@ -0,0 +1,92 @@
+# Aspect Meta Model RDF Vocabulary
```

Comment:
This should be a good title. For understanding the decision one has to read this ADR anyway.